src/script_rs/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "script_rs"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+rand = "0.8.5"
+rand_xoshiro = "0.6.0"
```

Comment:
Rust's stdlib does not include random number generators. `rand` provides a number of things, but not xoshiro generators -- `rand_xoshiro` is part of the rust-random org. The implementations are identical to David Blackman and Sebastiano Vigna's implementations (see linked code, other interesting things at https://prng.di.unimi.it/#intro).

I specifically chose xoshiro256++ as this is the default (since 1.7) generator used by Julia's `Random` stdlib. My objective was to replicate the PRNG behavior as closely as possible between the two. The implementations of Julia's xoshiro256++ and `rand_xoshiro`'s are identical, thus, the statistical properties are identical in that respect; the seeding procedure are slightly different (but both use OS entropy at their core), so that's one source of variation that can't conveniently be locked down. 
Additionally, Julia uses SIMD bulk generation with xoshiro256++ when filling arrays (i.e. for calls of the form `rand!(::AbstractArray{Float64})`) of bits types such as `Float64`, `UInt64`, etc. I am not certain that `rustc` will SIMD the loop body inside the `fill` call -- I'd need to examine the assembly; based on the source code, it does not seem likely unless some very aggressive inlining takes place. Note that it would be straightforward to guarantee if we use `portable_simd` and nightly Rust, but that would still not necessarily yield the same behavior as Julia's bulk generation.

The easy way to force the generated sequences to be identical is to provide identical initial states (256 bits, which Julia takes as 4 `UInt64`s and Rust as a static array of `UInt8`s), and for each `i=1:N, i=1:m` advance the state using the `jump` function. Additionally, to opt out of Julia's SIMD bulk generation, replace the `rand!` calls with something like
```julia
function elementwiserand!(rng, z::AbstractArray{T}) where {T}
    for i ∈ eachindex(z)
        z[i] = rand(rng, T)
    end
    z
end
```

The details above are superfluous for this purpose, but if one is going to benchmark sorting algorithms -- which can be sensitive to patterns (or lack thereof) in data -- it is worthwhile to consider such aspects.